src/main/resources/data.properties

```diff
@@ -66,4 +66,7 @@ identity.feature.toggle=false
 identity.realm=backbase
 identity.client=hybrid-flow
 # Health check
-healthcheck.use.actuator=true
\ No newline at end of file
+healthcheck.use.actuator=true
+# Some static external arrangement ids that we need to be ingested always
+# This is a comma (,) separated value
+arrangement.currentaccount.externalIds=1cdb2224-8926-4b4d-a99f-1c9dfbbb4699
```

Comment:
Does it have to be an `UUID` or can it be something like `transaction-pull-ingestion-1`?

Reply:
Right now this is what we have in the mock service. After changing it, we can change this one too.

Reply:
Done

Reply:
Discussed offline: this would make the number of arrangements created not match `numberOfArrangements` which could be confusing, so we're not doing it.